tools/create-element/generator/element.ts

```diff
@@ -259,6 +259,7 @@ export async function generateElement(options: GenerateElementOptions): Promise<
     return console.log('‼️ No package.json found.', '� Scaffold a repository first');
   } else {
     await writeElementFiles(options);
+    analyzeElement; // skip this for now, come back to fix later
```

Comment:
Was this intentional for this PR?

Reply:
Yeah it pacifies a linting warning sky unused cars. Probably should be an eslint disabled but 🤷‍♂️